src/AppBundle/Security/PersonalizationPolicyProvider.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace AppBundle\Security;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ConfigBuilderInterface;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Security\PolicyProvider\PolicyProviderInterface;
+
+class PersonalizationPolicyProvider implements PolicyProviderInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function addPolicies(ConfigBuilderInterface $configBuilder): void
+    {
+        $configBuilder->addConfig([
+            'personalization' => [
+                'show_menu_item' => null
```

Comment:
```suggestion
                'view' => null
```
I think it's more consistent with the rest of the application